tests/admin_views/tests.py

```diff
@@ -6166,3 +6167,70 @@ def test_implicitly_generated_pk(self):
         post_data = {'name': '2'}
         response = self.client.post(reverse('admin:admin_views_implicitlygeneratedpk_change', args=(1,)), post_data)
         self.assertEqual(response.status_code, 302)
+
+
+@override_settings(
+    PASSWORD_HASHERS=['django.contrib.auth.hashers.SHA1PasswordHasher'],
+    ROOT_URLCONF="admin_views.urls",
+)
+class SiteEachContextTest(TestCase):
```

Comment:
There has been discussion about better organizing the `admin_views` tests. How about moving this to a new file along with its admin and urlpatterns (if that works)? Something like `tests/admin_views/test_adminsite.py`